Makefile

```diff
@@ -297,7 +297,11 @@ $(YQ_BIN): $(YQ) ## Building yq from the tools folder
 
 .PHONY: lint
 lint: $(GOLANGCI_LINT) lint-latest ## Lint codebase
-	$(GOLANGCI_LINT) run -v
+	$(GOLANGCI_LINT) run -v $(GOLANGCI_LINT_EXTRA_ARGS)
+
+.PHONY: lint-fix
+lint-fix: $(GOLANGCI_LINT) ## Lint the codebase and run auto-fixers if supported by the linter
```

Comment:
could you please also add a note about this in https://capz.sigs.k8s.io/developers/development.html ?

Reply:
Done. 
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1987/files#diff-4d554fade8fefc047dcf883418cc72311cb5b62f3e0d20f7cbbfb849faa7335aR115